speech/tests/system.py

```diff
@@ -245,3 +247,41 @@ def test_stream_recognize_single_utterance(self):
             for results in self._make_streaming_request(
                     file_obj, single_utterance=False):
                 self._check_results(results.alternatives)
+
+    def test_long_running_recognize(self):
+
+        content = None
+        reps = 5
```

Comment:
Maybe I am n00b but I don't know what `reps` is supposed to mean. In the future, please use "full words" unless it's painfully clear. But, as you'll see below, I recommend dropping this variable all together.

Reply:
ok.